src/pybind/mgr/dashboard/frontend/src/app/shared/forms/cd-validators.ts

```diff
@@ -59,6 +59,18 @@ export class CdValidators {
     }
   }
 
+  /**
+   * Validator function in order to validate network addresses e.g 192.168.1.0/24.
+   * @returns {ValidatorFn} A validator function that returns an error map containing `pattern`
+   *   if the validation check fails, otherwise `null`.
+   */
+  static networkAddress(): ValidatorFn {
+    // prettier-ignore
+    const ipv4Rgx =
+      /^((25[0-5]|2[0-4][0-9]|[01]?[0-9][0-9]?)\.){3}(25[0-5]|2[0-4][0-9]|[01]?[0-9][0-9]?)(\/(3[0-2]|[12]?[0-9])?)$/i;
+    return Validators.pattern(ipv4Rgx);
+  }
```

Comment:
Recently @aaSharma14 started using [validator.js](https://www.npmjs.com/package/validator) to validate the urls rather than maintaining our own regex which is not stable and misses a few validation. I am afraid that could happen here too? validator.js provides a function validate ip address as well, [isIP()](https://github.com/validatorjs/validator.js/blob/master/src/lib/isIP.js#L47). 

But the problem is it doesn't validate CIDR and, as I understand it, you need CIDR. So why not using an alternative here like https://github.com/silverwind/is-cidr, which [then supports both v4 and v6](https://github.com/silverwind/is-cidr?tab=readme-ov-file#usage)?

Reply:
Using `is-cidr` package now instead of previous regex. 
Thanks for the tip!